File: Programmerare.CrsTransformations.Test/com/programmerare/testData/TestResult.java
package com.programmerare.com.programmerare.testData;

import com.programmerare.crsTransformations.CrsTransformationAdapter;

import java.util.List;

public class TestResult {
    // TODO: getters instead of public fields below ...
    public final CrsTransformationAdapter adapter;
    public final long totalNumberOfSecondsForAllTransformations;
    public final List<TestResultItem> testResultItems;

    TestResult(
            CrsTransformationAdapter adapter,
            long totalNumberOfSecondsForAllTransformations,
            List<TestResultItem> testResultItems
    ) {
        this.adapter = adapter;
        this.totalNumberOfSecondsForAllTransformations = totalNumberOfSecondsForAllTransformations;
        this.testResultItems = testResultItems;
    }
}

File: TestAndExampleProjects/Programmerare.CrsTransformations.Test/TestData/TestResult.cs
using System.Collections.Generic;

namespace Programmerare.CrsTransformations.TestData {

public class TestResult {
    public ICrsTransformationAdapter Adapter { get; }
    public long TotalNumberOfSecondsForAllTransformations { get; }
    public IList<TestResultItem> TestResultItems { get; }
    
    public TestResult(
        ICrsTransformationAdapter adapter,
        long totalNumberOfSecondsForAllTransformations,
        IList<TestResultItem> testResultItems
    ) {
        this.Adapter = adapter;
        this.TotalNumberOfSecondsForAllTransformations = totalNumberOfSecondsForAllTransformations;
        this.TestResultItems = testResultItems;
    }
}
}